src/pages/index.tsx

```diff
@@ -66,6 +69,52 @@ export default class IndexPage extends Component {
             null;
     }
 
+    public renderKeynotes(keynotes: any[]): ReactNode | null {
+        return keynotes && keynotes.length > 0 ?
+            <>
+                <div className="row mb-3">
+                    <h3 className="col text-center">Our Keynote Speakers</h3>
+                </div>
+                <div className="row justify-content-center">
+                    {
+                        keynotes
+                            .sort((a: any, b: any) => {
+                                const aTime = new Date(a.node.startDateTime);
+                                const bTime = new Date(b.node.startDateTime);
+                                if (aTime > bTime) { return 1; }
+                                if (aTime < bTime) { return -1; }
+
+                                return 0;
+                            })
+                            .map(({ node }: any, index: number) => (
+                                <div className="col-md-6 col-lg-5 col-sm-7 col-8 mb-3 mb-lg-0 keynote-speaker-card" key={ index }>
+                                    <div className="card mb-3 h-100">
+                                        <div className="row no-gutters h-100">
+                                            <div className="col-md-5 col-lg-4">
+                                                <img src={ node.speakers[0].photo.fixed.src } className="card-img h-100 w-100" alt={ node.speakers[0].name } />
+                                            </div>
+                                            <div className="col-md-7 col-lg-8">
+                                                <div className="card-body h-100 d-flex flex-column">
+                                                    <h5 className="card-title">{ node.speakers[0].name }</h5>
+                                                    <div className="flex-grow-1">
+                                                        <p className="font-italic mb-0">{ node.title }</p>
+                                                        <small className="text-muted">{ node.sessionType } &#64; { node.startTime }</small>
+                                                    </div>
+                                                    <Link to={`/2019/sessions#${ getUrlSafeId(node.title) }`}>
```

Comment:
Did you mean to have the year hard coded in this url?

Reply:
mfw this is a draft pr

Reply:
oh. github emailed me.